exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggTemplate.java

```diff
@@ -278,7 +281,7 @@ public void setup(HashAggregate hashAggrConfig, HashTableConfig htConfig, Fragme
   }
 
   @Override
-  public AggOutcome doWork() {
+  public AggOutcome doWork() throws SchemaChangeException {
```

Comment:
`doWork()` is used elsewhere as well. e.g. StreamingAggregator. Would it make sense to address other `doWork()` s in this PR. Better yet factor out another interface which the HashAggregator/StreamingAggregator implement - this would contain `public AggOutcome doWork() throws SchemaChangeException`